src/main/java/org/jitsi/videobridge/DtlsTransport.java

```diff
@@ -591,7 +597,7 @@ protected void consume(@NotNull PacketInfo packetInfo)
                                 packetInfo.getPacket().getBuffer(),
                                 packetInfo.getPacket().getOffset(),
                                 packetInfo.getPacket().getLength()));
-                    ByteBufferPool.returnBuffer(packetInfo.getPacket().getBuffer());
+                    packetDiscarded(packetInfo);
```

Comment:
Hm, I thought `packetDiscarded` would be more appropriate here but I've just realized it'll be misleading, so we _should_ return it to the pool directly.